Index.html

```diff
@@ -28,6 +28,204 @@ <h2 id="myTitle">Software Developer</h2>
             </p>
         </div>
     </section>
-    
+    <section class="myAbilities">
+        <img class="line" src="./assets/vectorLine.svg" alt="horizontal line">
+        <div class="skillsSection">        
+            <div class="mySkills">
+                <a id="languages" href="#"><h2>Languages</h2></a>
+                <ul class="abilities">
+                    <li>
+                        <a href="#">JavaScript</a>
+                    </li>
+                    <li>
+                        <a href="#">Ruby</a>
+                    </li>
+                    <li>
+                        <a href="#">Html</a>
+                    </li>
+                    <li>
+                        <a href="#">Html</a>
+                    </li>
+                </ul>
+            </div>
+            <div class="mySkills">
+                <a id="frameworks" href="#"><h2>Frameworks</h2></a>
+                <ul class="abilities">
+                    <li>
+                        <a href="#">Bootstrap</a>
+                    </li>
+                    <li>
+                        <a href="#">Ruby on Rails</a>
+                    </li>
+                    <li>
+                        <a href="#">RSPec</a>
+                    </li>
+                    <li>
+                        <a href="#">CapyBara</a>
+                    </li>
+                    <li>
+                        <a href="#">Selenium</a>
+                    </li>
+                </ul>
+            </div>
```

Comment:
- [ ] Great job for implementing this list of knowledge, but there is something to check out. As you can see this is actually a list and as it's a list it's supposed to be a `ul` tag instead of a `div`.
   - So for that it should be like this:
   ```HTML
   <ul class="knowledge">
      <li class="languages">
        <ul class="languages-list">
           <li class="language"></li>
           ....
       </ul> 
    </li>
    ....
   </ul>
   ```